dotcom-rendering/src/web/components/MostViewedFooterLayout.tsx

```diff
@@ -130,15 +118,11 @@ export const MostViewedFooter = ({ sectionName, format, ajaxUrl }: Props) => {
 						<Hide when="above" breakpoint="leftCol">
 							<h2 css={headingStyles}>Most popular</h2>
 						</Hide>
-						<Lazy margin={300}>
```

Comment:
Sorry - I can't work out what happens with this lazy margin + suspense, is it included somewhere else or not needed at all?